backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -379,6 +379,9 @@ public static String parseState(String s) {
   }
 
   public static Map<String, Boolean> parseSymptoms(String symptoms) {
+    if (symptoms == null) {
+      return Collections.emptyMap();
```

Comment:
If each of the symptoms are `false` should we also return `Collections.emptyMap()`? 🤔

Reply:
hmm, I don't think so, because all symptoms false doesn't technically mean empty